### ISSUE TITLE
gymnasium.error.NameNotFound: Environment highway doesn't exist. 

### ISSUE THREAD

**Issue by sunningshore** [eleurent/highway-env#431]
# gymnasium 0.27
import gymnasium as gym
env = gym.make("highway-v0")
I got this error: 
gymnasium.error.NameNotFound: Environment highway doesn't exist. 

Thanks.

**Comment by eleurent** [comment]
Yes, this is because gymnasium is only used for the development version yet, it is not in the latest release. But I'll make a new release today, that should fix the issue.

**Comment by sunningshore** [comment]
Seems still not solved, thanks.

**Comment by XiaoLu0223** [comment]
Seems still not solved, thanks.

**Comment by eleurent** [comment]
Sorry about that. 
While I figure out why this is broken, please use the following preamble:
```
import highway_env
highway_env.register_highway_envs()
```
this should solve the issue.
